src/schemas/21P-530/schema.js

```diff
@@ -34,6 +34,42 @@ let schema = {
   type: 'object',
   additionalProperties: false,
   definitions: {
+    address: {
+      type: 'object',
+      properties: {
+        street: {
+          type: 'string'
+        },
+        aptNum: {
+          type: 'string',
+          minLength: 1,
+          maxLength: 5
+        },
+        city: {
+          type: 'string'
+        },
+        state: {
+          type: 'string',
+          minLength: 2,
+          maxLength: 2
+        },
+        country: {
+          type: 'string',
+          minLength: 2,
+          maxLength: 2
+        },
+        postalCode1: {
```

Comment:
This seems limiting for something like a Canadian postal code, which is six characters, or other countries that might have different formats.